cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -700,14 +769,14 @@ void ImageFlip::disparityCallback(const stereo_msgs::DisparityImage::ConstPtr& d
 	cv::Mat rot33 = cv::Mat::eye(3,3,CV_64FC1);
 	for (int r=0; r<2; ++r)
 		for (int c=0; c<3; ++c)
-			rot33.at<double>(r,c) = rot_mat.at<double>(r,c);
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.